### PR TITLE
Add chapter for `Introduction` and `Terms and abbreviations`

### PR DESCRIPTION
**Pull request by kito-cheng**
Preview PDF:
[riscv-abi.pdf](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/files/7532419/riscv-abi.pdf)

### CODE REVIEW

File: introduction.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This specification uses the following terms and abbreviations.
+
+[width=80%]
+|===
+| Term  | Meaning
+
+| ABI   | Application Binary Interface.
+| gABI  | Generic System V Application Binary Interface.
+|===
```
Comment (cmuellner) [inline introduction.adoc:22]: Further possible entries:
ELF | Executable and Linking Format
psABI | processor specific ABI
DWARF | Debugging With Arbitrary Record Formats
GOT | Global Offset Table
PLT | Program Linkage Table
PC | Program Counter
TLS | Thread-Local Storage
NTBS | Null-terminated byte string

**Comment by kito-cheng** [comment]
Changes:
- Add more abbreviations per @cmuellner's suggesion.

File: introduction.adoc (hunk 1 above)
Comment (kito-cheng) [reply introduction.adoc:30]: Thanks!

File: introduction.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
+= Introduction
+
+This specification provides the processor-specific application binary interface
+document for RISC-V.
+
+This specification consists of the following three parts:
+
+- Calling convention
+- ELF specification
+- Dwarf specification
```
Comment (jrtc27) [inline introduction.adoc:10]:
```suggestion
- DWARF specification
```

File: introduction.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+= Terms and Abbreviations
+
+This specification uses the following terms and abbreviations.
```
Comment (jrtc27) [inline introduction.adoc:14]:
```suggestion
This specification uses the following terms and abbreviations:
```

File: introduction.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+| ELF   | Executable and Linking Format
+| psABI | processor specific ABI
+| DWARF | Debugging With Arbitrary Record Formats
+| GOT   | Global Offset Table
+| PLT   | Program Linkage Table
+| PC    | Program Counter
+| TLS   | Thread-Local Storage
+| NTBS  | Null-terminated byte string
+|===
+
```
Comment (jrtc27) [inline introduction.adoc:31]:
```suggestion
```

File: introduction.adoc [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+| ABI   | Application Binary Interface.
+| gABI  | Generic System V Application Binary Interface.
+| ELF   | Executable and Linking Format
+| psABI | processor specific ABI
+| DWARF | Debugging With Arbitrary Record Formats
+| GOT   | Global Offset Table
+| PLT   | Program Linkage Table
+| PC    | Program Counter
+| TLS   | Thread-Local Storage
+| NTBS  | Null-terminated byte string
```
Comment (jrtc27) [inline introduction.adoc:29]:
```suggestion
| ABI   | Application Binary Interface
| gABI  | Generic System V Application Binary Interface
| ELF   | Executable and Linking Format
| psABI | Processor-Specific ABI
| DWARF | Debugging With Arbitrary Record Formats
| GOT   | Global Offset Table
| PLT   | Program Linkage Table
| PC    | Program Counter
| TLS   | Thread-Local Storage
| NTBS  | Null-Terminated Byte String
```

**Comment by kito-cheng** [comment]
Changes:
- Apply @jrtc27's suggestions from code review

**Review by jrtc27 — APPROVED** [review]
(though please squash commits before merging)

**Pull request merged**